dv/uvm/core_ibex/scripts/ibex_sim.mk

```diff
@@ -0,0 +1,99 @@
+###############################################################################
+
+TB-COMPILE-STAMP = $(METADATA-DIR)/tb.compile.stamp
+rtl_tb_compile: $(METADATA-DIR)/tb.compile.stamp
+rtl-tb-compile-var-deps := SIMULATOR COV WAVES # Rebuild if these change
+
+rtl_sim_run: $(rtl-sim-logs)
+
+check_logs: $(comp-results)
+
+FCOV-STAMP = $(METADATA-DIR)/fcov.stamp
+riscv_dv_fcov: $(METADATA-DIR)/fcov.stamp
+
+MERGE-COV-STAMP = $(METADATA-DIR)/merge.cov.stamp
+merge_cov: $(METADATA-DIR)/merge.cov.stamp
+
+REGR-LOG-STAMP = $(METADATA-DIR)/regr.log.stamp
+collect_results: $(METADATA-DIR)/regr.log.stamp
+
+rtl-sim-logs +=
+comp-results +=
+
+###############################################################################
+# Compile ibex core TB
+#
+# Note that this doesn't depend on the seed: the DUT doesn't depend on which
+# test we're running!
+
+tb-compile-vars-path := $(BUILD-DIR)/.tb.vars.mk
+-include $(tb-compile-vars-path)
+tb-compile-vars-prereq = $(call vars-prereq,comp,compiling TB,$(rtl-tb-compile-var-deps))
+
+$(METADATA-DIR)/tb.compile.stamp: \
+  $(tb-compile-vars-prereq) $(all-verilog) $(all-cpp) $(risc-dv-files) \
+  scripts/compile_tb.py yaml/rtl_simulation.yaml \
+  | $(BUILD-DIR)
+	@echo Building RTL testbench
+	$(verb)env PYTHONPATH=$(PYTHONPATH) \
+	scripts/compile_tb.py \
+	  --dir-metadata $(METADATA-DIR)
+	$(call dump-vars,$(tb-compile-vars-path),comp,$(rtl-tb-compile-var-deps))
+	@touch $@
+
+###############################################################################
+# Run ibex RTL simulation with randomly-generated program and uvm stimulus
```

Comment:
If I've understood this correctly this step will also run the directed tests? If so the comment needs updating.